test/e2e/network/kube_proxy.go

```diff
@@ -229,7 +231,9 @@ var _ = SIGDescribe("Network", func() {
 		// Run the closewait command in a subroutine so it keeps waiting during postFinTimeoutSeconds
 		// otherwise the pod is deleted and the connection is closed loosing the conntrack entry
 		go func() {
-			framework.RunHostCmdOrDie(fr.Namespace.Name, "e2e-net-client", cmd)
+			defer ginkgo.GinkgoRecover()
+			_, err = framework.RunHostCmd(fr.Namespace.Name, "e2e-net-client", cmd)
+			framework.ExpectNoError(err)
```

Comment:
not sure if should check for the errors, if the test succeed it will exit and don't know how this is going to behave 🤷‍♂